modules/Core/classes/Avatars/UploadedImageAvatarSource.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class UploadedImageAvatarSource extends AvatarSourceBase {
+
+    public function __construct(Language $language) {
+        $this->_name = $language->get('admin', 'avatar_source_uploaded_image');
+        $this->_module = 'Core';
+        $this->_settings = ROOT_PATH . '/modules/Core/includes/admin_avatar_settings/uploaded_image.php';
+    }
+
+    public function get(User $user): ?string {
+        $base_url = ($this->_full_url ? rtrim(URL::getSelfURL(), '/') : '') . ((defined('CONFIG_PATH')) ? CONFIG_PATH . '/' : '/') . 'uploads/avatars';
+
+        if (Settings::get('custom_user_avatars') && $user->data()->has_avatar) {
+            $exts = ['png', 'jpg', 'jpeg'];
+
+            if ($user->hasPermission('usercp.gif_avatar')) {
+                $exts[] = 'gif';
+            }
+
+            foreach ($exts as $ext) {
+                if (file_exists(ROOT_PATH . "/uploads/avatars/{$user->data()->id}.{$ext}")) {
```

Comment:
Need to be a way to remove or unselect default custom avatar

People most likey want Custom avatar to be priority one, and minecraft next but if they have uploaded a custom default image then they are not able to fallback to minecraft avatar